### PR TITLE
chore: add issues templates for bugs and features

### PR DESCRIPTION
**Pull request by evegufy**
## Description

provide overarching issue templates for bugs and features

## Why

increase comfort and quality at issue creation

### CODE REVIEW

**Review by FaGru3n — APPROVED** [review]
LGTM and gives everyone the option to use the new default templates.

Only one excetion i see here now, with using lables like `bug`, `feature` this will expect in every repo the 2 labels. 

But in case they can be overwritten with manual templates for the product teams and their repos.

If they will be implemeted we have to inform everyone on mailing list or Matrix Chat.

**Review by Phil91 — APPROVED** [review]
LGTM, I do agree with @FaGru3n point. about the two expected labels in each repository

**Comment by evegufy** [comment]
@FaGru3n @Phil91 thanks for reviewing, `bug` and`enhancement` are [default labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#about-default-labels), so every repo has them as long nobody deletes them.

**Comment by evegufy** [comment]
@Phil91 @FaGru3n I just pushed some small improvement and will inform everyone on mailing list or Matrix Chat about the pr.

**Review by Phil91 — APPROVED** [review]

**Review by tunacicek — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Please don't include any personal or sensitive data.
+-->
+
+### Current Behavior:
+<!-- A concise description of what you're experiencing. -->
+
+### Expected Behavior:
+<!-- A concise description of what you expected to happen. -->
+
+### Steps To Reproduce:
```
Comment (florianrusch-zf) [inline .github/ISSUE_TEMPLATE/bug.md]: As these are headlines, I would basically drop the colon (`:`).

```suggestion
### Current Behavior
<!-- A concise description of what you're experiencing. -->

### Expected Behavior
<!-- A concise description of what you expected to happen. -->

### Steps To Reproduce
```

**Review by florianrusch-zf — APPROVED** [review]
LGTM, just a minor thing ;-)

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Please fill out this template as good as possible.
+Please don't include any personal or sensitive data.
+-->
+
+### Current Behavior
+<!-- A concise description of what you're experiencing. -->
+
+### Expected Behavior
+<!-- A concise description of what you expected to happen. -->
+
```
Comment (mhellmeier) [inline .github/ISSUE_TEMPLATE/bug.md:18]: Open for discussion: Should we add a headline like `### System Information` or something similar? Especially for bugs, I think it is important to know the system where the bug occurs (operating system, versions, etc.).

Comment (evegufy) [reply .github/ISSUE_TEMPLATE/bug.md:18]: @mhellmeier I could add [here](https://github.com/eclipse-tractusx/.github/pull/19/files#diff-b577c75bfe1a70d8f33d404b7aa81aff373147db26fe6d5a54283c3dea107f83R22) operating system explicitly, other infos, like version, are already part of the list.

Comment (mhellmeier) [reply .github/ISSUE_TEMPLATE/bug.md:18]: Would also be fine for me.

Comment (evegufy) [reply .github/ISSUE_TEMPLATE/bug.md:18]: done

**Review by SebastianBezold — APPROVED** [review]
👍

**Review by mhellmeier — APPROVED** [review]

**Review by matbmoser — APPROVED** [review]
Nice templates!

**Pull request merged**